eqt/ui/UIFormWidget.py

```diff
@@ -458,7 +458,7 @@ def saveAllWidgetStates(self):
         '''
         self.widget_states = self.getAllWidgetStates()
 
-    def getWidgetStates(self):
+    def getSavedWidgetStates(self):
```

Comment:
as above, add alias and deprecation warning

Reply:
done